sklearn/metrics/_scorer.py

```diff
@@ -127,6 +127,44 @@ def __init__(self, score_func, sign, kwargs):
         self._score_func = score_func
         self._sign = sign
 
+    @staticmethod
+    def _check_pos_label(pos_label, classes):
+        if pos_label not in list(classes):
+            raise ValueError(
+                f"pos_label={pos_label} is not a valid label: {classes}"
+            )
+
+    def _select_proba(self, y_pred, classes, support_multi_class):
+        """Select the column of y_pred when probabilities are provided."""
```

Comment:
We could note that if `support_multi_class=True` and `y_pred` is valid, this method is a noop.